docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -464,6 +464,17 @@
                 <tr>
                     <th colspan="3" class="section" id="task"><a href="../api/overview.html">Task configuration</a></th>
                 </tr>
+                <tr>
+                    <td class="property" id="job.debounce.time.ms">job.debounce.time.ms</td>
+                    <td class="default"> 2000 </td>
+                    <td class="description">
+                        How long the Leader processor will wait before performing some notification based action (like JobModel recalculation) from the moment the notification came.
```

Comment:
Should be sufficient, I was thinking about LeaderElection, but is using the same event anyway.
Will change.